lib/webtask.js

```diff
@@ -383,14 +387,6 @@ exports.ensure_authorized = function (admin, ctx) {
     };
 };
 
-var lru; // LRU cache of token revocation information
-
-exports.clear_revocation_lru = function (jti) {
```

Comment:
Now we don't have a `clear_revocation_lru` exposed anywhere. Do we still need it?